### DISCUSSION TITLE
Featured in PC Pro magazine

### DISCUSSION THREAD

**Discussion by B-Hartley** [benleb/surepy#68] [Show and tell]
Did you know you were featured in PC Pro magazine for this project?
https://subscribe.pcpro.co.uk/

If you haven't seen it, I can send you a pic.

**Comment by benleb** [comment]
Oh wow! Cool! Thank you very much!! 🤝  I didn't and probably never would have seen it otherwise :)

I wonder if I can buy one here in Germany... 🤔  If not, could you send me one from the UK? :D

Thanks @B-Hartley!
